docs/source/troubleshooting.rst

```diff
@@ -147,9 +147,9 @@ Use the Web Auth API
 ********************
 
 Converse supports the `Web Authentication API <https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API>`_
-which let's it use the secure credential management of the browser to get the
-uesr credentials to automatically log the user in. This however requires that
-the user saves his or her username and password in the browser. Often the user
+which uses the secure credential management of the browser to get the
+uesr credentials to automatically log the user in. This requires
```

Comment:
Thanks. `user` is still misspelled as `uesr`, can you please fix that as well?

Reply:
Done, thanks @jcbrand . Also did some light copy editing for readability if that seems okay.

Reply:
Definitely, thanks, I just found one typo.